.github/workflows/build.yml

```diff
@@ -140,103 +140,122 @@ jobs:
-          docker tag "$DOCKER_IMAGE_ID" "$DOCKER_IMAGE_ID:$VERSION"
-          docker push "$DOCKER_IMAGE_ID:$VERSION"
-          docker tag "$DOCKER_IMAGE_ID:with-browser" "$DOCKER_IMAGE_ID:$VERSION-with-browser"
-          docker push "$DOCKER_IMAGE_ID:$VERSION-with-browser"
-          # We also want to tag the latest stable version as latest
-          if [[ ! "$VERSION" =~ (RC|rc) ]]; then
-            echo "Publish to Docker Hub as $DOCKER_IMAGE_ID:latest"
-            docker tag "$DOCKER_IMAGE_ID" "$DOCKER_IMAGE_ID:latest"
-            docker push "$DOCKER_IMAGE_ID:latest"
-            docker tag "$DOCKER_IMAGE_ID:with-browser" "$DOCKER_IMAGE_ID:latest-with-browser"
-            docker push "$DOCKER_IMAGE_ID:latest-with-browser"
-          fi
-      - name: Build loadimpact/k6
-        run: docker build -t $LI_DOCKER_IMAGE_ID --target legacy .
-      - name: Publish loadimpact/k6:master image to Docker Hub
-        if: ${{ github.ref == 'refs/heads/master' }}
-        run: |
-          echo "Publish to Docker Hub as $LI_DOCKER_IMAGE_ID:master"
-          docker tag "$LI_DOCKER_IMAGE_ID" "$LI_DOCKER_IMAGE_ID:master"
-          docker push "$LI_DOCKER_IMAGE_ID:master"
-      - name: Publish loadimpact/k6 tagged version image to Docker Hub
-        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
-        run: |
-          VERSION="${VERSION#v}"
-          echo "Publish to Docker Hub as $LI_DOCKER_IMAGE_ID:$VERSION"
-          docker tag "$LI_DOCKER_IMAGE_ID" "$LI_DOCKER_IMAGE_ID:$VERSION"
-          docker push "$LI_DOCKER_IMAGE_ID:$VERSION"
+          echo "Publish to Docker Hub as $DOCKER_IMAGE_ID:$VERSION and $LI_DOCKER_IMAGE_ID:$VERSION"
+          docker buildx build --push \
+            --platform linux/amd64,linux/arm64 \
+            -t $DOCKER_IMAGE_ID:$VERSION .
+          docker buildx build --push \
+            --target with-browser \
+            --platform linux/amd64,linux/arm64 \
+            -t $DOCKER_IMAGE_ID:$VERSION-with-browser .
+          # LoadImpact images are deprecated, we don't build arm64 for it.
+          docker buildx build --push \
+            --platform linux/amd64 \
+            -t $LI_DOCKER_IMAGE_ID:$VERSION .
           # We also want to tag the latest stable version as latest
           if [[ ! "$VERSION" =~ (RC|rc) ]]; then
-            echo "Publish to Docker Hub as $LI_DOCKER_IMAGE_ID:latest"
-            docker tag "$LI_DOCKER_IMAGE_ID" "$LI_DOCKER_IMAGE_ID:latest"
-            docker push "$LI_DOCKER_IMAGE_ID:latest"
+            echo "Publish to Docker Hub as $DOCKER_IMAGE_ID:latest and $LI_DOCKER_IMAGE_ID:latest"
+            docker buildx build --push \
+              --platform linux/amd64,linux/arm64 \
+              -t $DOCKER_IMAGE_ID:latest .
+            docker buildx build --push \
+              --target with-browser \
+              --platform linux/amd64,linux/arm64 \
+              -t $DOCKER_IMAGE_ID:latest-with-browser .
+            docker buildx build --push \
```

Comment:
nit: can we have the same comment about loadimpact images being deprecated here as well.